cluster/gce/configure-vm.sh

```diff
@@ -430,31 +442,46 @@ EOF
   fi
 }
 
+function try-download-release() {
+  # TODO(zmerlynn): Now we REALLy have no excuse not to do the reboot
+  # optimization.
+
+  # TODO(zmerlynn): This may not be set yet by everyone (GKE).
```

Comment:
Will this case will also be hit when launching released versions (e.g. running `get.k8s.io | bash`)?